ml_tools/eolearn/ml_tools/train_test_split.py

```diff
@@ -27,8 +27,10 @@ class TrainTestSplitType(Enum):
 class TrainTestSplitTask(EOTask):
     """Randomly assign each pixel or groups of pixels to multiple subsets (e.g., test/train/validate).
 
-    Input pixels are defined by an input feature (e.g., MASK_TIMELESS with polygon ids, connected component ids, or
-    similar), that groups together pixels with similar properties.
+    When sampling PER_PIXEL the input feature defines the shape of the output feature. For PER_CLASS and PER_VALUE the
```

Comment:
> When sampling PER_PIXEL the input feature defines the shape of the output feature. 

Is that not true for all 3 strategies?

Reply:
will specify that in that case it ONLY defines the shape